pipeline_inspection_metacontrol/pipeline_inspection_metacontrol/system_modes_bridge.py

```diff
@@ -0,0 +1,40 @@
+import rclpy
+from rclpy.node import Node
+
+from system_modes_msgs.srv import ChangeMode
+from mros2_msgs.srv import NodeMode
+
+class BridgeService(Node):
+    def __init__(self):
+        super.__init__('System_mode_bridge')
+        self.srv = self.create_service(
+                NodeMode,
+                '/ros_reasoner/change_node_mode',
+                self.change_mode_cb)
+
+    def change_mode_cb(self, request, response):
+        system_modes_cli = self.create_client(
+                ChangeMode,
+                '/' + request.node_name + '/change_mode',
+                callback_group=ReentrantCallbackGroup()) 
+        try:
+            req = ChangeMode.Request()
+            req.mode_name = request.mode_name
```

Comment:
The NodeMode srv doesnt have a req.mode_name and req.node_name because they are inverted in the srv definition in the mros2_msgs package

Reply:
This is a ChangeMode srv though, that is correct right?